inc/saiswitch.h

```diff
@@ -500,6 +500,9 @@ typedef enum _sai_switch_attr_t
        Default no map */
     SAI_SWITCH_ATTR_QOS_TC_AND_COLOR_TO_DSCP_MAP,
 
+    /* Enable switch shell [bool] (default to false) */
+    SAI_SWITCH_ATTR_SWITCH_SHELL_ENABLE,
```

Comment:
How about clubbing all this as part of a saidebug module. This may have all logger/debug functionality and everything else needed for debugging. Also are the shell commands etc supposed to be vendor specific and more like HW debug?